.github/PULL_REQUEST_TEMPLATE.md

```diff
@@ -69,3 +69,20 @@ This change added tests and can be verified as follows:
   - If yes, how is the feature documented? (not applicable / docs / GoDocs / not documented)
   - If a feature is not applicable for documentation, explain why?
   - If a feature is not documented yet in this PR, please create a followup issue for adding the documentation
+
+### Matching PR in forked repository
+
+PR in forked repository: <!-- ENTER URL HERE -->
+
+<!--
+After opening this PR, the build in apache/pulsar-client-go will fail and instructions will
+be provided for opening a PR in the PR author's forked repository.
```

Comment:
We didn't add this restriction to the pulsar-client-go repo. For non-committer contributors, it's required by the ASF that their PRs' CI must be approved by a committer before running the CI.

Reply:
Got it.

Reply:
Done.